framework/Objects/Meta_Fields.php

```diff
@@ -0,0 +1,266 @@
+<?php
+
+namespace JustCoded\WP\Framework\Objects;
+
+/**
+ * Meta_Fields is the base class for acquiring data from custom fields.
+ */
+
+class Meta_Fields {
+
+	/**
+	 * Plugin used for extending of post custom fields.
+	 *
+	 * Available values:
+	 *      'just-custom-fields'
+	 *      'advanced-custom-fields'
+	 *
+	 * @var string
+	 */
+	public $custom_fields_plugin = 'just-custom-fields';
+
+	/**
+	 * Current $post object
+	 *
+	 * @var \WP_Post
+	 */
+	public $post;
+
+	/**
+	 * Current $term object
+	 *
+	 * @var \WP_Term
+	 */
+	public $term;
```

Comment:
$post and $term properties are related to specific class realization. you can't add them here.

however getter funcs check for object. so we can define property $object - and store post or term meta their.

each child class will define object property type inside (wp post or wp_term)

Reply:
done